source/css/scss-common/base/_forms.scss

```diff
@@ -721,7 +721,8 @@ textarea {
 
 // country code
 .a-input-countryCode {
-  max-width: 55px;
+  max-width: 62px;
+  text-align: right;
```

Comment:
But  Linn kamilla has metioned that the textbox must grow as you type. So basically the UX folks want to expand the button based on the content. So probably we need to add some javascript?